src/main/resources/templates/admin/moduleView.html

```diff
@@ -0,0 +1,535 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org">
+<head th:replace="~{/admin/header :: head}"/>
+<body>
+
+<span th:replace="~{/admin/header :: top}"/>
+
+<a href="/admin/">Home</a>
+-
+<a href="/admin/modules">Modules</a>
+
+<th:block th:if="${canManageModules}">
+    <h2 th:text="|Module ${module.name}|">Module (name)</h2>
```

Comment:
Might be good to also include the module ID here since names aren't required to be unique (and probably won't be since we create new modules for module content edits).